server_run.py

```diff
@@ -30,6 +31,17 @@ def server_run(args):
     set_scoped_session(scoped_session(sessionmaker(bind=engine),
                                       scopefunc=lambda: _request_ctx_stack.top))
 
+    if args.recreate_db:
+        if args.debug:
+            exit("Not recreating the with `--debug`, "
+                 "which usually leads to multiple restarts on file change.\n"
+                 "This would override any database change every time.")
+        print("Dropping pycroft model schema")
+        drop_db_model(engine)
```

Comment:
@mueslo It is quite a problem that `server_start.py` doesn't work from scratch. It is a useful thing for the one, central script we use all the time to start pycroft to be able to create the schema if it doesn't exist.
If you say `run_server.py` shouldn't touch the database, then I suggest renaming it or making a different script that itself may invoke `run_server.py` (e.g. `manage.py`).

Since you always start without the schema, running the app fails. This shouldn't be the case, I think it is perfectly valid for the app to at least bootstrap the schema.